src/user/userRoutes.ts

```diff
@@ -1,10 +0,0 @@
-import { Router } from 'express';
-import { register, login, verifyEmail } from './userController';
-
-const router = Router();
-
-router.post('/register', register);
-router.post('/login', login);
-router.post('/verify-email', verifyEmail);
```

Comment:
is this protected route or not?

Reply:
@OmarMuhtaseb i dont think it should be protected since the flow is like this:
user registers -> gets verification email -> clicks on the link to verify email which will hit this route. all while not having a jwt generated, so i dont think it makes sense to generate a jwt for an unauthenticated user. wdyt?